src/Integration.Vsix/Telemetry/SonarLintIntegrationPackage.IOleCommandTarget.cs

```diff
@@ -1,77 +0,0 @@
-﻿//-----------------------------------------------------------------------
-// <copyright file="SonarLintIntegrationPackage.IOleCommandTarget.cs" company="SonarSource SA and Microsoft Corporation">
-//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
-//   Licensed under the MIT License. See License.txt in the project root for license information.
-// </copyright>
-//-----------------------------------------------------------------------
-using Microsoft.VisualStudio;
-using Microsoft.VisualStudio.OLE.Interop;
-using Microsoft.VisualStudio.Shell;
-using System;
-using System.Diagnostics;
-using OLEConstants = Microsoft.VisualStudio.OLE.Interop.Constants;
-
-namespace SonarLint.VisualStudio.Integration.Vsix
-{
-    /// <summary>
-    /// Partial class to provide handling for SQM commands.
-    /// </summary>
-    [ProvideMenuResource("Menus.ctmenu", 1)]
-    partial class SonarLintIntegrationPackage : IOleCommandTarget
-    {
-        private SonarLintSqmCommandTarget sqmCommandHandler;
-
-        private void InitializeSqm()
-        {
-            this.sqmCommandHandler = new SonarLintSqmCommandTarget();
-            
-            // It's possible that one of the command line packages has already initialized
-            // the facade. However, calling it multiple times won't cause an error.
-            SonarLintSqmFacade.Initialize(this);
```

Comment:
All of these changes should be reverted to, except for line 30 as the SqmFacade does need to be initialised by MEF.
As the comment says, it doesn't matter if the facade is initialised multiple times, but it doesn't need to be initialised here.